platforms/documentation/docs/src/docs/userguide/dep-man/dependency-resolution/artifact_transforms.adoc

```diff
@@ -98,15 +103,24 @@ include::sample[dir="snippets/dependencyManagement/artifactTransforms-simple/kot
 include::sample[dir="snippets/dependencyManagement/artifactTransforms-simple/groovy",files="build.gradle[tags=artifact-transform-registration]"]
 ====
 
-=== 4. Use the Transformed Artifacts
+"To" attributes are used to supply describe the format or type of the artifacts that this transform can use as an input, and "from" attributes to describe the format or type of the artifacts that it produces as an output.
+
+=== 4. Use the Transform
+
+During a build, Gradle *automatically* runs registered transforms to satisfy a resolution request if a match is not directly available.
 
-During a build, Gradle uses registered transforms to produce a required artifact if it’s not directly available.
+Since no variants exist supplying the requested format (as none contain the `artifactType` attribute with a value of `"transformed-jar"`), Gradle attempts to construct a chain of transformations that will supply it.
```

Comment:
```suggestion
Since no variants exist supplying artifacts of requested format (as none contain the `artifactType` attribute with a value of `"transformed-jar"`), Gradle attempts to construct a chain of transformations that will supply it.
```